problems/02.md

```diff
@@ -12,20 +12,20 @@ It's a good idea to keep [the hypercore README](https://github.com/mafintosh/hyp
 
 To get started, create a _package.json_ with all the dependencies you will need for the workshop, including hypercore. In a terminal window, enter:
 
-```
-git clone https://gist.github.com/okdistribute/6d2d11e43977054d0613f4714cc48076 kappa-workshop
+```sh
+$ git clone https://gist.github.com/okdistribute/6d2d11e43977054d0613f4714cc48076 kappa-workshop
 ```
 
 Then, use your terminal to change the current working directory to the `kappa-workshop`:
 
-```
-cd kappa-workshop
+```sh
+$ cd kappa-workshop
```

Comment:
having the prefix `$` makes it harder to copy-paste no? at least if it's not added as a css attribute e.g. `code pre:before { text: '$';}` or similar

Reply:
Ah good catch!